### ISSUE TITLE
fan speed support in web interface

### ISSUE THREAD

**Issue by mbdrlp** [geoffdavis/esphome-mitsubishiheatpump#160]
Hi,

I am not able to display the fan speed list in the web interface. How can I activate it?
And I would like to set the fan speed through the WEB API but I don't find how to do that. I did it for temperature and mode, and it works well. Only fan speed is missing.

Thanks

**Comment by Injineers** [comment]
i know this is old ~ but @mbdrlp  did you enable the features on the card?

![Image](https://github.com/user-attachments/assets/dc9f957b-0447-4a26-938f-c454ec7a6354)

**Comment by mbdrlp** [comment]
Yes this is issue is solved. I switched from Jeedom to Home Assistant  😂
